runners/spark/src/main/java/org/apache/beam/runners/spark/io/ConsoleIO.java

```diff
@@ -31,17 +32,27 @@ private ConsoleIO() {
 
   /**
    * Write on the console.
+   *
```

Comment:
It would suggest either keeping the old `Print to console.`  or going with  `Write to console.`